### PR TITLE
WI #1579 #1590 Fix hover and completion exceptions

### PR DESCRIPTION
**Pull request by LANOYDO**
see issues https://github.com/TypeCobolTeam/TypeCobol/issues/1579 and https://github.com/TypeCobolTeam/TypeCobol/issues/1590

### CODE REVIEW

File: TypeCobol/Compiler/Nodes/Node.cs [hunk 1]
```diff
@@ -821,6 +821,7 @@ private void Dump(StringBuilder str, int i)
         /// <returns>Correpsonding DataDefinition</returns>
         public DataDefinition GetDataDefinitionFromStorageAreaDictionary(StorageArea searchedStorageArea, bool? isReadDataDefiniton=null)
```
Comment (smedilol) [inline TypeCobol/Compiler/Nodes/Node.cs]: Can you add attribute `[CanBeNull]` before StorageArea searchedStorageArea ?
This way Resharper will know that the value can be null.

File: TypeCobol/Compiler/Nodes/Node.cs [hunk 2]
```diff
@@ -848,6 +849,7 @@ public DataDefinition GetDataDefinitionFromStorageAreaDictionary(StorageArea sea
         
         public DataDefinition GetDataDefinitionForQualifiedName(QualifiedName qualifiedName, bool? isReadDictionary=null)
```
Comment (smedilol) [inline TypeCobol/Compiler/Nodes/Node.cs]: Same here:
Can you add attribute [CanBeNull] before QualifiedName qualifiedName ?
This way Resharper will know that the value can be null.

File: TypeCobol/Compiler/Nodes/Node.cs (hunk 1 above)
Comment (LANOYDO) [reply TypeCobol/Compiler/Nodes/Node.cs]: done
https://github.com/TypeCobolTeam/TypeCobol/pull/1595/commits/59139c835bd4751fb47886a0d3305206f1272b62

File: TypeCobol/Compiler/Nodes/Node.cs (hunk 2 above)
Comment (LANOYDO) [reply TypeCobol/Compiler/Nodes/Node.cs]: done
https://github.com/TypeCobolTeam/TypeCobol/pull/1595/commits/59139c835bd4751fb47886a0d3305206f1272b62

**Pull request merged**